int32 type
float64 step
float64 lateral
float64 turn

int32 TASK_NONE = 0
int32 TASK_WALK = 1
int32 TASK_KICK = 2
